index.html

```diff
@@ -1632,41 +1632,76 @@ <h4>Threat model</h4>
         </ul>
     </section>
     <section>
-      <h3>Cross-border online age restricted sales</h3>
+      <h3>Boarding an intercontinental flight - holder binding</h3>
       <h4>Background</h4>
       <p>
-        Anna is French and has an interest in good whiskeys. She decides to buy a whiskey from a highly touted
-        producer and online retailer based in Ireland called TradeWhiskey. Using her eID (electronic Identity),
-        Anna can attest with a predetermined required level of assurance that she is of legal drinking age and get the
-        product shipped to France.
+        Anna is preparing for her Singapore Airlines flight from Paris to Singapore. To travel to Singapore Anna needs a valid passport.
+        Therefore, she has to present her boarding pass along with her passport at the gate. 
+        Anna has heard about the new option at Charles de Gaulle airport to use a digital version of her passport (eID) for boarding and decides to try it out.
+
       </p>
       <h4>Distinction</h4>
       <p>
-        This use case is challenging because it has to be compliant with government regulations, which require a certain
-        assurance that Anna is the rightful holder and subject of the presented eID.
-        For this, Anna has to be remotely authenticated as the rightful holder of the eID that attests her legal age.
+This use case is challenging because it has to be compliant with government regulations which requires a certain
+assurance that Anna is the rightful holder and subject of the presented eID.
+For this Anna has to be authenticated as the rightful holder of the eID.
```

Comment:
```suggestion
This use case is challenging because it has to be compliant with government regulations which require a certain
assurance that Anna is the rightful holder and subject of the presented eID.
For this, Anna has to be authenticated as the rightful holder of the eID.
```